src/mahoji/commands/chop.ts

```diff
@@ -229,7 +225,7 @@ export const chopCommand: OSBMahojiCommand = {
 			iQty: options.quantity ? options.quantity : undefined,
 			powerchopping: powerchop,
 			forestry: forestry_events,
-			twitchers: twitchers_gloves || 'egg',
+			twitchers: twitchers_gloves as TwitcherGloves,
```

Comment:
This should definitely not need a typecast. You have it typed wrong in the options above, probably.